CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+### 0.0.2
```

Comment:
I'd recommend getting rid of this file and just having all the changes associated in the release section for the repository. This seems like a redundant file to me

Reply:
Makes sense, will remove in favor of using Github's Release functionality to track version-level changes